README.md

```diff
@@ -80,18 +80,20 @@ colcon build
 
 ## Run
 
-The `TURTLEBOT3_MODEL` environment variable must be set when running the simulation application (not needed for robot application). Valid values are `burger`, `waffle`, and `waffle_pi`.
+The `TURTLEBOT3_MODEL` environment variable must be set when running both robot and simulation application. Valid values are `burger`, `waffle`, and `waffle_pi`.
 
 Launch the application with the following commands:
 
 - *Running Robot Application on a Robot*
     ```bash
+    export TURTLEBOT3_MODEL=<robot-model>
     source robot_ws/install/local_setup.sh
     roslaunch cloudwatch_robot deploy_rotate.launch
     ```
 
 - *Running Robot Application in a Simulation*
     ```bash
+    export TURTLEBOT3_MODEL=waffle_pi
```

Comment:
It says `export TURTLEBOT3_MODEL=<robot-model>` for the robot application. Let's make it consistent.

Reply:
I thought on a real robot, it should depend on the real robot, but not for simulation.

Reply:
Ideally, we have this as optional (like how it is for ROS2 CloudWatch and ROS1/ROS2 Hello World sample apps), and so this README will be structured in a way that's consistent with the other sample apps.

Reply:
I'll make the required changes to make `TURTLEBOT3_MDOEL` optional

Reply:
Let's forget about making the setting of the environment variable optional.

However, let's move this example line out to the same position as it is in the other READMEs. Writing it this way makes it seem like `TURTLEBOT3_MODEL` can take on different values between the robot and simulation applications.

Reply:
Done